src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+$second-text-color: #616070;
+$button-color: #00acdc;
+
+.card {
+  margin: 0;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__photo {
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__name--position {
+    margin: 0 16px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-text-color;
+  }
+
+  &__code--position {
+    margin: 4px 16px 0 ;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rate--position {
+    margin: 16px 16px 0;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price--position {
+    margin: 24px 16px 0;
+  }
+
+  &__buy {
+    height: 40px;
+    color: #fff;
```

Comment:
I would recommend you to visit:
https://shorturl.at/dsOWZ

As it seems that the styles order standards are violated